var/spack/repos/builtin/packages/eztrace/package.py

```diff
@@ -6,21 +6,77 @@
 from spack import *
 
 
-class Eztrace(AutotoolsPackage):
+class Eztrace(Package):
     """EZTrace is a tool to automatically generate execution traces
        of HPC applications."""
 
-    homepage = "https://eztrace.gforge.inria.fr"
-    url      = "https://gitlab.com/eztrace/eztrace/-/archive/eztrace-1.1-10/eztrace-eztrace-1.1-10.tar.gz"
+    homepage = "https://gitlab.com/eztrace"
     maintainers = ['trahay']
+    git = "https://gitlab.com/eztrace/eztrace.git"
 
-    version('1.1-10', sha256='97aba8f3b3b71e8e2f7ef47e00c262234e27b9cb4a870c85c525317a83a3f0d4')
+    version('master',  branch='master')
+    version('2.0-rc1', sha256='2ad322df5aeb0c668ebadbfeb3b21c8841831b2c15944d86d1d217f738fee74e')
+    version('1.1-13', sha256='6144d04fb62b3ccad41af0268cd921161f168d0cca3f6c210c448bb0b07be7e0')
+    version('1.1-10', sha256='63d1af2db38b04efa817614574f381e7536e12db06a2c75375d1795adda3d1d8')
```

Comment:
Why did the hash change for 1.1-10? This is bad a from reproducibility/security point of view.

Reply:
I changed eztrace tags from `eztrace-1.1-10` to `1.1-10` on the git repo. This prevent gitlab from generating tarballs named `eztrace-eztrace-1.1-10`, but this changes the hash :(
I can add the `eztrace-1.1-10` tag to eztrace gitlab repo, which should recreate a tarball with  the same hash, but in this case, the `url_for_version` function would become messy.

Reply:
OK that's a good enough explanation for me. Keep the new hash. Thanks!